gdnative-core/src/nativescript/user_data.rs

```diff
@@ -559,13 +559,13 @@ mod local_cell {
             match self {
                 LocalCellError::DifferentThread { original, current } => write!(
                     f,
-                    "Accessing from the wrong thread, expected {:?} found {:?}",
+                    "accessing from the wrong thread, expected {:?} found {:?}",
                     original, current
                 ),
                 LocalCellError::BorrowFailed => write!(
                     f,
-                    "Borrow failed; a &mut reference was requested, but one already exists. Cause is likely a re-entrant call \
-                    (e.g. a GDNative Rust method calls to GDScript, which again calls a Rust method on the same object)."
+                    "borrow failed; a &mut reference was requested, but one already exists. cause is likely a re-entrant call \
+                    (e.g. a gd-native rust method calls to gdscript, which again calls a rust method on the same object)."
```

Comment:
"GDNative", "Rust" and "GDScript" are proper nouns. They should still be capitalized. The "lowercase" in the guideline only means that "you should not capitalize the first word as if it's at the start of a sentence", I believe.

Reply:
IIRC the guideline doesn't specify what happens if there are multiple sentences like here. Probably they implicitly suggest "don't do that", but here it clearly makes sense to guide the user a bit, and having `;` or `-` instead of full stop would hardly be better.

Reply:
Those were good points. I went ahead and changed the proper nouns back as of a5a6f99.

Thank you for the quick response on those changes!